tests/run-make/compile-stdin/rmake.rs

```diff
@@ -0,0 +1,13 @@
+// When provided standard input piped directly into rustc, this test checks that the compilation
+// completes successfully and that the output can be executed.
+//
+// See <https://github.com/rust-lang/rust/pull/28805>.
+
+//@ ignore-cross-compile
+
+use run_make_support::{run, rustc};
+
+fn main() {
+    rustc().arg("-").stdin("fn main() {}").run();
```

Comment:
I really like the `stdin` helper, it's so much nicer to read than `echo '...' | ` imo

Reply:
Me too. So convenient :D